tests/extensions/config/utils.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+def _get_and_check_houston_configs(key=None, value=None):
```

Comment:
This is very much a query but I understood that in PEP8 the _ prefix was meant to indicate : Local and only to be called within this file.
This clearly does not follow that rule, any reason?

Reply:
This has been fixed.  I wasn't intending to use those outside of the file but moved them later to a utils file.